src/routes/routes.tsx

```diff
@@ -16,12 +17,18 @@ const router = createBrowserRouter([
         index: true,
         element: <StartScreen />,
       },
-      ...machineLearningExamples.map((example) => {
-        return {
-          path: `/machineLearning/${example.name}`,
-          element: <Example name={example.name} />,
-        };
-      }),
+      {
+        path: '/machineLearning/example1',
+        element: <CertificatesTable />,
```

Comment:
I still would preffer that you have separate component for the view itself. And that component is using thann CertificatesTable. Like "Example1View.tsx", and than there render CertificatesTable. Example1View will for render the title "Example 1" and table component separatelly. List of the certfificattes harcoded in the view and passed to the table as prop. So the table component takes certs and renders them in a table, nothing else. This way you are using benefit of react separating things in components. Later View will be in charge for some more logic regarding certificates and not only rendering table